lib/qa/authorities/getty/aat.rb

```diff
@@ -23,7 +29,7 @@ def sparql(q)
                  skos:inScheme <http://vocab.getty.edu/aat/> ;
                  gvp:prefLabelGVP [skosxl:literalForm ?name].
               FILTER regex(?name, \"#{search}\", \"i\") .
-            } LIMIT 10"
```

Comment:
The limit was in place to keep the query from taking too long to execute. Why are you removing it?